Code/Source/Entity/Terrain/TerrainEntityManagerEditorSystemComponent.h

```diff
@@ -30,10 +30,9 @@ namespace RGL
         static void GetProvidedServices(AZ::ComponentDescriptor::DependencyArrayType& provided);
         static void GetIncompatibleServices(AZ::ComponentDescriptor::DependencyArrayType& incompatible);
         static void GetRequiredServices(AZ::ComponentDescriptor::DependencyArrayType& required);
-        static void GetDependentServices(AZ::ComponentDescriptor::DependencyArrayType& dependent);
```

Comment:
Why did you removed this one?

Reply:
There was no point in overriding the method when its body was empty. Removed this to reduce clutter.